securedrop/journalist_templates/admin.html

```diff
@@ -3,7 +3,10 @@
 <h1>{{ gettext('Admin Interface') }}</h1>
 
 <form action="{{ url_for('admin.add_user') }}">
-  <button type="submit" id="add-user"><i class="fa fa-plus"></i>{{ gettext('ADD USER') }}</button>
+  <button type="submit" id="add-user">
+    <img src="{{ url_for('static', filename='icons/plus.png') }}" class="icon" width="13" height="15" alt="plus-icon">
```

Comment:
This would be an exception to my request to translate alt tags. Where the icon is a visual repetition of adjacent text, we should use `alt=""`, per [WCAG recommendations for decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).

Reply:
`alt=""` used where the alt text does not add information relative to what the adjacent text already provides.